docs/2.10.0/docs/canton/usermanual/contract_keys.rst

```diff
@@ -278,8 +278,8 @@ Caveats to keep in mind are:
   However, this is easy to change.
```

Comment:
add a short explanation why delegation does not work as stated further above, as the delegated parties may not see the Keyed contract.

Reply:
Done.